crates/bevy_render/src/texture/mod.rs

```diff
@@ -29,7 +39,9 @@ impl Plugin for ImagePlugin {
             feature = "dds",
             feature = "tga",
             feature = "jpeg",
-            feature = "bmp"
+            feature = "bmp",
+            feature = "basis-universal",
+            feature = "ktx2",
```

Comment:
should this have a `feature = "dds'` line?

Reply:
There is one just above. It already existed because the image crate has some dds decompression support.

Reply:
Hahaha oops. Carry on :)